cmd/boot-script-service/oauth.go

```diff
@@ -143,32 +203,38 @@ func QuoteArrayStrings(arr []string) []string {
 // Returns the OAuthClient struct containing the client ID, secret, etc. as well
 // as the access token and an error if one occurred.
 func (client *OAuthClient) RequestClientCreds() (accessToken string, err error) {
-	var (
-		url  string
-		resp []byte
-	)
+	return client.FetchAccessToken(oauth2AdminBaseURL + "/admin/clients")
+	// log.Printf("Attempting to register OAuth2 client")
+	// debugf("Sending request to %s", url)
+	// resp, err = client.CreateOAuthClient(url)
+	// if err != nil {
+	// 	err = fmt.Errorf("Failed to register OAuth2 client: %v", err)
+	// 	debugf("Response: %v", string(resp))
+	// 	return
+	// }
+	// log.Printf("Successfully registered OAuth2 client")
+	// debugf("Client ID: %s", client.Id)
 
-	url = oauth2AdminBaseURL + "/admin/clients"
-	log.Printf("Attempting to register OAuth2 client")
-	debugf("Sending request to %s", url)
-	resp, err = client.CreateOAuthClient(url)
-	if err != nil {
-		err = fmt.Errorf("Failed to register OAuth2 client: %v", err)
-		debugf("Response: %v", string(resp))
-		return
-	}
-	log.Printf("Successfully registered OAuth2 client")
-	debugf("Client ID: %s", client.Id)
+	// url = oauth2AdminBaseURL + "/oauth2/auth"
+	// log.Printf("Attempting to authorize OAuth2 client")
+	// debugf("Sending request to %s", url)
+	// _, err = client.AuthorizeOAuthClient(url)
+	// if err != nil {
+	// 	err = fmt.Errorf("Failed to authorize OAuth2 client: %v", err)
+	// 	debugf("Response: %v", string(resp))
+	// 	return
+	// }
+	// log.Printf("Successfully authorized OAuth2 client")
 
-	url = oauth2PublicBaseURL + "/oauth2/token"
-	log.Printf("Attempting to fetch token from authorization server")
-	debugf("Sending request to %s", url)
-	accessToken, err = client.PerformTokenGrant(url)
-	if err != nil {
-		err = fmt.Errorf("Failed to fetch token from authorization server: %v", err)
-		return
-	}
-	log.Printf("Successfully fetched token")
+	// url = oauth2PublicBaseURL + "/oauth2/token"
+	// log.Printf("Attempting to fetch token from authorization server")
+	// debugf("Sending request to %s", url)
+	// accessToken, err = client.PerformTokenGrant(url)
+	// if err != nil {
+	// 	err = fmt.Errorf("Failed to fetch token from authorization server: %v", err)
+	// 	return
+	// }
+	// log.Printf("Successfully fetched token")
 
 	return
```

Comment:
We can probably get rid of this return if this function is just returning the result of `client.FetchAccessToken`. We can also probably delete the comments.